CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Display default placeholder on the Product page (#5088)
+- Removed `visibilitychange` eventlistener on microcart beforeDestroy hook
```

Comment:
Add ID of the issue. Bestly, with the link to the issue.

Reply:
Done
Sorry for delay)